web-ui/src/components/pending_skills/PendingSkillsCard.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useContext, useEffect, useState } from "react";
+
+import { getSkillMembers } from "../../api/memberskill";
+import { AppContext } from "../../context/AppContext";
+
+import { Avatar, Chip, Card, CardHeader, CardContent } from "@material-ui/core";
+
+const PendingSkillsCard = ({ pendingSkill }) => {
+  const { state } = useContext(AppContext);
+  const { selectMemberProfileById } = AppContext;
+  const { csrf, memberProfiles } = state;
+
+  const { description, id, name } = pendingSkill;
+  const [members, setMembers] = useState([]);
+
+  useEffect(() => {
+    const getMembers = async () => {
+      let res = await getSkillMembers(id, csrf);
+      const copy = [...members];
+      if (res && res.payload && res.payload.data) {
+        res.payload.data.map((m) => {
+          const { memberid } = m;
+          const member = selectMemberProfileById(state)(memberid);
```

Comment:
Don't select these here, keep the IDs instead and select the profiles from context where you use them. This gets rid of the need to keep the profiles in context and you can fix the `eslint-disable-next-line` below